FiveCalls/FiveCalls/MyImpactViewController.swift

```diff
@@ -58,7 +66,27 @@ class MyImpactViewController : UITableViewController {
         streakLabel.text = weeklyStreakMessage(for: weeklyStreakCount)
         let numberOfCalls = viewModel.numberOfCalls
         impactLabel.text = impactMessage(for: numberOfCalls)
-        subheadLabel.isHidden = numberOfCalls == 0
+        if (numberOfCalls == 0) {
+            subheadLabel.isHidden = true
+            subheadLabel.addConstraint(NSLayoutConstraint(item: subheadLabel, attribute: .height, relatedBy: .equal, toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: 0))
+        }
+        
+        signInContainer.layer.masksToBounds = false
+        signInContainer.layer.shadowColor = UIColor.black.cgColor
+        signInContainer.layer.shadowOffset = CGSize(width: 1.0, height: 1.0)
+        signInContainer.layer.shadowOpacity = 0.2
+        signInContainer.layer.shadowRadius = 1.2
+
+        profileContainer.layer.masksToBounds = false
+        profileContainer.layer.shadowColor = UIColor.black.cgColor
+        profileContainer.layer.shadowOffset = CGSize(width: 1.0, height: 1.0)
+        profileContainer.layer.shadowOpacity = 0.2
+        profileContainer.layer.shadowRadius = 1.2
```

Comment:
This shadow setup is identical. I'd favor moving all of this into a UIView subclass.

Something like `ProfileCard`? We can move all this common code there (as well as the runtime corner radius attribute in the Storyboard.

Then we can update both IBOutlets to be of that class:

<img width="410" alt="image" src="https://user-images.githubusercontent.com/59140/34962046-bb7a5e3a-fa07-11e7-9748-1239dfb48288.png">

Reply:
Also, for the shadow offset: `CGSize(width:0, height: 1)`  (light source from top)